domains/kuroigg.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "kuroigg",
+        "email": "mrlangdon197@gmail.com"
+    },
+    "record": {
+        "CNAME": "kuroigg.github.io"
```

Comment:
please add more stuff to your website, not just "Hello World" text

Reply:
About that, its currently work in progress, and its just a placeholder.